Makefile

```diff
@@ -75,13 +75,20 @@ integration-test-with-docker: prepare-docker-runner-image generate-env-file ## R
 		${DOCKER_BUILDER_IMAGE_NAME} \
 		make integration-test
 
+
+.PHONY: publish-to-rubygems
+publish-to-rubygems:
```

Comment:
You could add a comment like our other methods so it comes up when you run `make`